directory-manifest/mvn-x3f-manifest.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+echo Calling (quietly) to make manifest.md for your file (resource) manifest
+
+rem batch file for invoking maven to get a directory listing for a current directory
+
+set HERE=%cd%
+set HEREPATH=file:///%HERE:\=/%
+
+set APPLICATION_DIR=%~dp0
+
+call mvn --quiet -f %APPLICATION_DIR%../pom.xml exec:java -Dexec.mainClass="com.xmlcalabash.drivers.Main" -Dexec.args="-omd=%HEREPATH%/manifest.md -ohtml=NUL %APPLICATION_DIR%/directory-manifest.xpl path=%HEREPATH%"
```

Comment:
This should address @galtm's feedback around proper path handling.

```suggestion
call mvn --quiet -f "%APPLICATION_DIR%../pom.xml" exec:java -Dexec.mainClass="com.xmlcalabash.drivers.Main" -Dexec.args="-omd="%HEREPATH%/manifest.md" -ohtml=NUL "%APPLICATION_DIR%/directory-manifest.xpl" path="%HEREPATH%"
```

Reply:
It's interesting that it worked the broken way (figures). Thanks AJ!

Reply:
I wouldn't do that but I hear it is sometimes done.